commerce-cart-content-web/src/main/resources/content/Language.properties

```diff
@@ -10,6 +10,7 @@ commerce-cart-web-portlet-instance-configuration-name=Commerce Cart Content
 commerce-order-items=Commerce Order Items
 commerce-order-items-help=Represents the list of commerce order items that is displayed.
 edit-cart=Edit Cart
+view-more=View More
```

Comment:
Hey @luca-pellizzon, shouldn't this key be inside the `commerce-lang` module instead? Or do you guys have a specific pattern for which keys go where that this follows?

Reply:
Hi Ethan, I 've put them near other similar labels. Let's see if @marco-leo want to move it elsewhere.